sql-cli/sql_cli/constants.py

```diff
@@ -3,7 +3,7 @@
 CONFIG_FILENAME = "configuration.yml"
 
 DEFAULT_ENVIRONMENT = "default"
-DEFAULT_AIRFLOW_HOME = f".airflow/{DEFAULT_ENVIRONMENT}"
+DEFAULT_BASE_AIRFLOW_HOME = ".airflow/"
```

Comment:
```suggestion
DEFAULT_BASE_AIRFLOW_HOME = ".airflow"
```
should work too, and makes it consistent with the other paths.